meetups/uk/meetups.md

```diff
@@ -68,10 +68,13 @@
         
 * [London Clojurians](https://www.meetup.com/London-Clojurians/)
     * Weekly meetup to learn Clojure  
+
+* [London Software Craftmanship](https://www.meetup.com/london-software-craftsmanship/)
```

Comment:
Thanks a good one. I know the name continues to have some controversy as there's a preference for a more inclusive name. However, inspite of this I still feel it's a valuable community to know about.